cms/templates/index.html

```diff
@@ -485,6 +500,54 @@ <h3 class="title">${_('Create Your First Library')}</h3>
       </div>
       %endif
 
+      % if is_programs_enabled:
+        % if len(programs) > 0:
+          <div class="programs programs-tab">
+            <!-- Classes related to courses are intentionally reused here, to duplicate the styling used for course listing. -->
+            <ul class="list-courses">
+              % for program in programs:
+              <li class="course-item">
+
+                <!-- TODO: Use the program ID contained in the dict to link to the appropriate view in the authoring app. -->
+                <a class="program-link" href="#">
+                  <h3 class="course-title">${program['name'] | h}</h3>
+
+                  <div class="course-metadata">
+                    <span class="course-org metadata-item">
+                      <!-- As of this writing, programs can only be owned by one organization. If that constraint is relaxed, this will need to be revisited. -->
+                      <span class="label">${_("Organization:")}</span> <span class="value">${program['organizations'][0]['key'] | h}</span>
+                    </span>
+                  </div>
+                </a>
+
+              </li>
+              % endfor
+            </ul>
+          </div>
+
+        % else:
+          <div class="notice notice-incontext notice-instruction notice-instruction-nocourses list-notices programs-tab">
+            <div class="notice-item has-actions">
+
+              <div class="msg">
+                <h3 class="title">${_("You haven't created any programs yet.")}</h3>
+                <div class="copy">
+                  <p>${_("Programs are groups of courses related to a common subject.")}</p>
+                </div>
+              </div>
+
+              <ul class="list-actions">
+                <li class="action-item">
+                  <!-- TODO: Link to the program creation view in the authoring app. -->
+                  <a href="#" class="action-primary action-create new-button action-create-program new-program-button"><i class="icon fa fa-plus icon-inline"></i> ${_('Create Your First Program')}</a>
```

Comment:
We are using ``button`` now instead of ``a`` for elements that are buttons. This is primarily for a11y although it also just makes sense and means you don't need the fake ``href="#"``.